spec/support/with_queue_adapter.rb

```diff
@@ -0,0 +1,18 @@
+module WithQueueAdapter
```

Comment:
@hackmastera Is this change needed?  I would have expected the change in environment/test.rb to just do this.

Reply:
It was a remnant of my roundabout path to this solution. Removed!